.changeset/gold-boats-wash.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/integration': patch
+---
+
+Expose an `annotationPrefix` property for SCM integrations
```

Comment:
I'm personally a bit unsure about this one - what the use cases will be, whether their values really will line up anyway, etc. There may be fairly little value in having it, and risk for confusion, where one slug is actually slightly different from another one etc, for example where azure might have that tricky third field etc. Thoughts @andrewthauer ?

Reply:
> where one slug is actually slightly different from another one etc, for example where azure might have that tricky third field etc.

@freben - I'm not sure what exactly what you mean here. My thinking is these would be the well known annotation for each SCM provider (e.g. `[scm-provider]/project-slug`). Rather then having the annotation key as a constant that is possibly in a plugin, it feels more naturely they live on the ScmIntegration itself?

Reply:
@freben further thoughts? or suggestions on a better direction for this change?

Reply:
None specific here, but if this adding the prefixes to the integration package are a possible concern, then I guess it would be safer to put them back into the `catalog-import` package for the time being? It could always be refactored later on once the possible usage becomes more clear.

Reply:
@andrewthauer @freben reverted my changes to `@backstage/integration`. Let me know if there's any another concerns I should address